src/app-gocardless/services/gocardless-service.js

```diff
@@ -17,17 +17,12 @@ import jwt from 'jws';
 import { SecretName, secretsService } from '../../services/secrets-service.js';
 
 const GoCardlessClient = nordigenNode.default;
-const goCardlessClient = new GoCardlessClient({
-  secretId: secretsService.get(SecretName.nordigen_secretId),
-  secretKey: secretsService.get(SecretName.nordigen_secretKey),
-});
-
-secretsService.onUpdate(SecretName.nordigen_secretId, (newSecret) => {
-  goCardlessClient.secretId = newSecret;
-});
-secretsService.onUpdate(SecretName.nordigen_secretKey, (newSecret) => {
-  goCardlessClient.secretKey = newSecret;
-});
+
+const getGocardlessClient = () =>
+  new GoCardlessClient({
+    secretId: secretsService.get(SecretName.nordigen_secretId),
+    secretKey: secretsService.get(SecretName.nordigen_secretKey),
+  });
```

Comment:
Small amount of refactors here. Just cleaning up the code by reducing the complexity.